packages/inventory/src/Stocktake/DetailView/ContentArea.tsx

```diff
@@ -1,21 +1,86 @@
-// import React, { FC } from 'react';
-// import { DataTable, useTranslation } from '@openmsupply-client/common';
-// import { useCustomerRequisitionColumns } from './columns';
-// import { useCustomerRequisitionLines } from '../api';
+import React, { FC } from 'react';
+import {
+  DataTable,
+  useTranslation,
+  Box,
+  Switch,
+  useIsGrouped,
+  useSortBy,
+  getDataSorter,
+} from '@openmsupply-client/common';
+import { useStocktakeColumns } from './columns';
+import { useStocktakeLines, useStocktakeItems } from '../api';
+import { StocktakeSummaryItem, StocktakeLine } from '../../types';
 
-// export const ContentArea: FC = () => {
-//   const { lines, onChangeSortBy, sortBy, pagination } =
-//     useCustomerRequisitionLines();
-//   const columns = useCustomerRequisitionColumns({ sortBy, onChangeSortBy });
-//   const t = useTranslation('common');
+const Expand: FC<{ rowData: StocktakeSummaryItem | StocktakeLine }> = ({
+  rowData,
+}) => {
+  return (
+    <Box p={1} height={300} style={{ overflow: 'scroll' }}>
+      <Box
+        flex={1}
+        display="flex"
+        height="100%"
+        borderRadius={4}
+        bgcolor="#c7c9d933"
+      >
+        <span style={{ whiteSpace: 'pre-wrap' }}>
+          {JSON.stringify(rowData, null, 2)}
+        </span>
+      </Box>
+    </Box>
+  );
+};
 
-//   return (
-//     <DataTable
-//       pagination={{ ...pagination, total: lines.length }}
-//       columns={columns}
-//       data={lines}
-//       onChangePage={pagination.onChangePage}
-//       noDataMessage={t('error.no-items')}
-//     />
-//   );
-// };
+const useStocktakeRows = (isGrouped: boolean) => {
+  const { sortBy, onChangeSortBy } = useSortBy<
+    StocktakeLine | StocktakeSummaryItem
+  >({
+    key: 'itemName',
+  });
+  const { data: lines } = useStocktakeLines();
+  const { data: items } = useStocktakeItems();
+
+  const rows = isGrouped
+    ? items?.sort(
+        getDataSorter(sortBy.key as keyof StocktakeSummaryItem, !!sortBy.isDesc)
+      )
```

Comment:
TODO: This should be using the `getColumnSorter` but I had an issue with it that I'll need to address